src/styles/sidebar/components/menu.scss

```diff
@@ -23,6 +23,10 @@
     transition: 0.2s ease-out;
     color: var.$color-text;
   }
+  &[aria-expanded='true'],
```

Comment:
Attention: non-button change, technically. This is to help give on states to menu toggles.